app/services/session.js

```diff
@@ -17,6 +17,11 @@ export default Ember.Service.extend({
     return store.peekAll("user_profile").get("firstObject") || null;
   }).volatile(),
 
+  userDefaultPrinter: function() {
+    const store = this.get("store");
+    return store.peekRecord("user", this.get("currentUser.id")).get("printer");
```

Comment:
```suggestion
    return this.get("store").peekRecord("user", this.get("currentUser.id")).get("printer");
```

No need of storing store to a variable.